package.json

```diff
@@ -23,5 +23,8 @@
   },
   "scripts": {
     "glow": "glow check"
+  },
+  "dependencies": {
```

Comment:
we could probably push this into the api package.json at `services/api/package.json` in case it's of use elsewhere (and avoid adding dependencies to too many packages if we can avoid it?

Reply:
Good call, updated now.